examples/gosoline-fixture-loading/main.go

```diff
@@ -0,0 +1,105 @@
+//+build fixtures
+
+package main
+
+import (
+	"github.com/applike/gosoline/pkg/application"
+	db_repo "github.com/applike/gosoline/pkg/db-repo"
```

Comment:
please try not to add these as an alias, they are not required. Also look around in all files that have been touched for unnecessary aliases